docs/howto/secrets-backend/google-cloud-secret-manager-backend.rst

```diff
@@ -26,7 +26,15 @@ a secret backend and how to manage secrets.
 Before you begin
 """"""""""""""""
 
-`Configure Secret Manager and your local environment <https://cloud.google.com/secret-manager/docs/configuring-secret-manager>`__, once per project.
+Before you start, make sure you have performed the following tasks:
+
+1.  Include sendgrid subpackage as part of your Airflow installation
+
+    .. code-block:: bash
+
+        pip install airflow[google]
```

Comment:
It should be 
```
airflow`[sengrid]`
```
but pre-commit is sad.

> ```
> rst ``code`` is two backticks.....................................................................................................Failed
> - hook id: rst-backticks
> - exit code: 1
> 
> docs/howto/secrets-backend/google-cloud-secret-manager-backend.rst:35:        pip install `airflow[google]`
> 
> ```

Reply:
```suggestion
        pip install 'apache-airflow[google]'
```

Reply:
I can't add `because pre-commit has a problem with this, but I updated the package name.